packages/block-library/src/gallery/v1/edit.js

```diff
@@ -299,7 +299,7 @@ function GalleryEdit( props ) {
 	function getImagesSizeOptions() {
 		const resizedImageSizes = Object.values( resizedImages );
 		return map(
-			filter( imageSizes, ( { slug } ) =>
+			imageSizes.filter( ( { slug } ) =>
```

Comment:
Defaults to an array:

https://github.com/WordPress/gutenberg/blob/a4040e40965b5430874413e5d5389ac889ad13b9/packages/block-editor/src/store/defaults.js#L133